### ISSUE TITLE
Mod compatability issue: Dragon strike and breath attack on dragons and hippogryphs can target Minecolonies citizens [ICE AND FIRE VERSION 2.1.9]

### ISSUE THREAD

**Issue by warning-djkajslh** [Alex-the-666/Ice_and_Fire#4238]
**NO SUGGESTIONS AT ALL**
This is not a repository for suggestions or issues not pertaining to the mod's code or gameplay.

**Describe the bug**
You can tell from the name that, this happened when I tried to defend the town from raids or when I tried to ultilize both dragons and Minecolonies guards in an adventure.

**To Reproduce**
1. Skip to the part where Minecol guard and dragon are following you
2. Get on tamed dragon and press the strike button while aiming at guards
3. They get damaged and angered at the dragon.

**Expected behavior**
I think it is safe to make it so that Minecolonies's citizens and visitors should be a blacklisted mob for dragon strike attack.
Similar to Thaumcraft 6 golems back in 1.12.2

**Screenshots**

**Additional context**

**Comment by FlameyFaucet** [comment]
Tamed mobs do not blacklist damage from each other. If you had a wolf and smacked a colonist or guard, the wolf will attack and if the target is a guard they will retaliate. This is natural.

**Comment by warning-djkajslh** [comment]
This isn't a smacking issue, it is a special immune option, try using the dragon strike attack against some tamed wolves while riding the dragon, you will see they are unharmed, the dragon strike attack is supposed to ignore all tamed mob, but guards do not work the same way.

**Comment by FlameyFaucet** [comment]
Citizens and guards do not qualify as "tamed", so if you want to go super-literate they should not be added to the list of immune mobs. This would also cause some issues where you'd want to intentionally damage another person's citizens or guards, but you can't, because they were included in the blacklist.

**Comment by warning-djkajslh** [comment]
Yes, they aren't classified as "tamed" since they are humans. In my case I would like Alex to make an exception for a mob to be completely ignored by the attack. Furthermore, I don't think you are understand what the attack is like, basically I would still be able to damage them if I punch them myself, and my dragon does also, only in a specific case that is mentioned aboe where citizens will not be harmed. Finally, why would I ever want to punch my citizens?

**Comment by FlameyFaucet** [comment]
Again, this will create issues for people who will want to intentionally hurt another person’s citizens, guards, whatever. 

If you don’t want dragons to hurt your guards, just...get farther away. This should not have to be something that the devs fix. The bite hitbox is also really _not that big_, it will require deliberate aimjng at your own citizens to damage them with it.

**Comment by warning-djkajslh** [comment]
Then the mod will have to register the colony owner instead of just black listing, still, I am sticking to this issue

**Comment by warning-djkajslh** [comment]
Then the mod will have to register the colony owner instead of just black listing, still, I am sticking to this issue
